packages/libs/lambda-at-edge/tests/default-handler/default-handler.test.ts

```diff
@@ -186,6 +199,35 @@ describe("Lambda@Edge", () => {
 
         expect(response.status).toEqual("200");
       });
+      
+      it("handles preview mode", async () => {
+        const event = createCloudFrontEvent({
+          uri: "/preview",
+          host: "mydistribution.cloudfront.net",
+          requestHeaders: {
+            cookie: [
+              {
+                key: "Cookie",
+                value: "__next_preview_data=abc; __prerender_bypass=def"
+              }
+            ]
+          }
+        });
+
+        if (trailingSlash) {
```

Comment:
You shouldn't need the `trailingSlash` block here, I added a redirect test where you can add `/preview` as one of the tests, it will test both kinds of redirects

Reply:
You mean changing the URI in `createCloudFrontEvent` to be either `/preview` or `/preview/` depending on whether `trailingSlash` is true, and then just testing for the expected response?

Reply:
Yep, basically if trailingSlash is true then you can pass `/preview/` as the URI, otherwise it is `/preview`.

Then you can add the same path `/preview` as a test case in redirect test, it will auto append trailing slash and test the redirects

Reply:
Done.

Reply:
Also i think you can add `/preview` to redirect test (around line 158).

Reply:
Done.